### DISCUSSION TITLE
auto completion and $TEXINPUT/$BIBINPUT

### DISCUSSION THREAD

**Discussion by jonas-frey** [James-Yu/LaTeX-Workshop#3466] [Q&A] [answered]
I'm using latex-workshop on kubuntu 22.04.

I have noticed that autocompletion (ie suggestions via dropdown menu) only works if I include my macro and bib files via absolute path, not when I give filenames relative to the paths stored in the $TEXINPUT/$BIBINPUT environment variables. Is there a way to make latex-workshop find my macro and bib files even if the names have to be dereferenced using the environment variables?

**Comment by James-Yu** [comment, marked as answer]
The extension does not consider $TEXINPUT/$BIBINPUT. Only absolute path and relative one to your root tex file are considered. I’m not sure if this is a popular feature request.
